pkg/api/defaults.go

```diff
@@ -148,15 +148,23 @@ func (cs *ContainerService) setOrchestratorDefaults(isUpgrade, isScale bool) {
 		switch o.KubernetesConfig.ContainerRuntime {
 		case Docker:
 			if o.KubernetesConfig.MobyVersion == "" || isUpdate {
-				if isUpdate && o.KubernetesConfig.MobyVersion != DefaultMobyVersion {
-					log.Warnf("Moby will be upgraded to version %s\n", DefaultMobyVersion)
+				if o.KubernetesConfig.MobyVersion != DefaultMobyVersion {
+					if isUpgrade {
+						log.Warnf("Moby will be upgraded to version %s\n", DefaultMobyVersion)
+					} else if isScale {
+						log.Warnf("Any new scaled up nodes will have Moby version %s\n", DefaultMobyVersion)
```

Comment:
Nitpick: I think `newly scaled-up` is more correct, or just say `Any new nodes...`.

Reply:
How about "new, scaled-up" ?

Reply:
Yeah, I like "new nodes". The user already knows it's a scale operation.